procedures/automation-android-setup.md

```diff
@@ -0,0 +1,200 @@
+        4. `sudo vi /etc/hiera/secrets.eyaml`
+        5. Look for the equivalent `fenix` secrets (Use the `/` to search, then type "fenix", then "enter", hitting `n` each time you want to step forward) and place your new products secrets in the same way
+            * For the two Google Play credentials, you may need to paste the encrypted secret in an IDE and space-indent it to the same level so it matches the indentation of the other Google Play credentials in the file
+        6. Save (`:x`, enter) to save the file
+        7. Disconnect from the puppet master
+
+    3. In `modules/signing_scriptworker`
+        1. You should've received signing credentials from step 1. Print out the decrypted file you received: `gpg -d <file from step 1>`
+        2. With the output, find the "prod creds" section, and copy the line where the second array item ends in "_dep" (this is the dep autograph config)
+        3. Edit `templates/dep-passwords-mobile.json.erb`. Add a new scope section in the format `project:mobile:$product:releng:signing:cert:dep-signing`
+            1. Paste the dep autograph config (remove the trailing comma, if any)
+            2. Replace the second item in that list you pasted so that, instead of having the autograph username, it has `<%= scope.function_secret(["autograph_$product_dep_username"]) %>` (so it fetches from `hiera`)
+            3. Replace the third item in that list you pasted so that, instead of having the autograph password, it has `<%= scope.function_secret(["autograph_$product_dep_password"]) %>` (so it fetches from `hiera`)
+        4. Edit `templates/passwords-mobile.json.erb`. Add a new scope section in the format `project:mobile:$product:releng:signing:cert:release-signing`
+            1. Paste the prod autograph config (remove the trailing comma, if any)
+            2. Replace the second item in that list you pasted so that, instead of having the autograph username, it has `<%= scope.function_secret(["autograph_$product_prod_username"]) %>` (so it fetches from `hiera`)
+            3. Replace the third item in that list you pasted so that, instead of having the autograph password, it has `<%= scope.function_secret(["autograph_$product_prod_password"]) %>` (so it fetches from `hiera`)
+        5. Edit `manifests/settings.pp`, adding the new scope prefix `project:mobile:$product:releng:signing:` to the `scope_prefixes` property of both `mobile-dep` and `mobile-prod`
+
+    4. In `modules/pushapk_scriptworker`
+        1. From step 1, you should have received two certificates (one for dep, and one for prod). They start with `---BEGIN CERTIFICATE---` and end with `---END CERTIFICATE---`, and were probably sent in the gpg-encrypted text file with the autograph credentials. For each of these, copy them, remove any indentation they may have, and put them both in the `files` directory of `pushapk_scriptworker` with the names `$product_dep.pem` and `$product_release.pem`
```

Comment:
Nit: let's add a note to say that the certificate is public, even though it may have been sent encrypted.